### PR TITLE
[Meshmodel] Restructure file system for `meshmodels`.

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes
1. Restructures filesystem hierarchy for meshmodels.
2. Imporve comp generation script.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
Dependent PR: https://github.com/meshery/meshkit/pull/339

File: scripts/component_generation/main.go [hunk 1]
```diff
@@ -565,16 +565,16 @@ func Spreadsheet(srv *sheets.Service, sheetName string, spreadsheet chan struct
...
-					Values: values,
-				}
-				response2, err := srv.Spreadsheets.Values.Append(spreadsheetID, sheetName, row).ValueInputOption("USER_ENTERED").InsertDataOption("INSERT_ROWS").Context(context.Background()).Do()
-				values = make([][]interface{}, 0)
-				batchSize = 100
-				if err != nil || response2.HTTPStatusCode != 200 {
-					fmt.Println(err)
-					continue
-				}
+				// row := &sheets.ValueRange{
```
Comment (MUzairS15) [inline scripts/component_generation/main.go:568]: This is done for testing. Will be removed.

File: server/meshmodel/helper.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ import (
 )
 
 var ArtifactHubComponentsHandler = meshmodel.ArtifactHub{} //The components generated in output directory will be handled by kubernetes
+var ModelsPath = "../meshmodel"
```
Comment (leecalcote) [inline server/meshmodel/helper.go:20]: This is different than this line, right? https://github.com/meshery/meshery/pull/8372/files#diff-76d1b6fe5a5ca5822c004882b83a214e655a9e59311faf7411411d6fc2f04787R63

File: scripts/component_generation/main.go (hunk 1 above)
Comment (leecalcote) [reply scripts/component_generation/main.go:568]: Gotcha

**Pull request merged**